pages/index.js

```diff
@@ -1,10 +1,11 @@
 import ArtGenerator from "@/components/ArtGenerator";
+import CanvasOne from "@/components/CanvasOne";
 import Wrapper from "@/components/Wrapper";
 
 export default function HomePage() {
   return (
     <Wrapper>
-      <ArtGenerator />
+      <CanvasOne />
```

Comment:
Please rename your canvas component to `Canvas`. In the future you can use the same component multiple times and then the number doesn't make sense any more.

Reply:
I renamed the component! :)

Thanks for the review! :)